### DISCUSSION TITLE
Osu classic skin cursor gets out of sync with pause cursor after alt-tab

### DISCUSSION THREAD

**Discussion by NoahFreelove** [ppy/osu#16262] [Q&A]
I noticed when alt-tabbing out of the game and going back in the pause cursor was out of sync with the player cursor.

Before alt-tab:
![image](https://user-images.githubusercontent.com/90488068/147451625-5063fb47-b461-42c0-97f3-4ec79094531f.png)

After alt-tab:
![image](https://user-images.githubusercontent.com/90488068/147451201-d91c3f96-ffa7-4dee-8d28-76b43d3af0d0.png)

To be clear, this is a very small thing. Just thought I'd bring it up if it is an issue.

**Comment by peppy** [comment]
I don't get it. What is your expectation? One is a static sprite and one is animated.

**Reply by bdach** [reply]
I checked this previously actually, they actually both animate, and the animation does desync on tabbing out.

**Reply by NoahFreelove** [reply]
Both are animated. The player cursor is out of sync with the pause cursor after an alt-tab. I apologize if the screenshot did not depict that clearly. Here is a video

Before alt-tab:
![BeforeAltTab](https://user-images.githubusercontent.com/90488068/147732706-c6296563-a6f5-4f04-8107-d0398346cb51.gif)

After alt-tab:
![AfterAltTab](https://user-images.githubusercontent.com/90488068/147732721-f1bf112f-d473-4428-8eb6-de1cdd1385d3.gif)

**Reply by peppy** [reply]
Probably just disable the animation then.
